Description: Fixed -v and -i and other flags in skill/snice
 There was a too greedy option parser
 Patch based on submitted patch by Yoshio Nakamura
Bug-Debian: http://bugs.debian.org/331419
Bug-Debian: http://bugs.debian.org/569030
Author: Craig Small <csmall@debian.org>

--- a/skill.c
+++ b/skill.c
@@ -464,6 +464,8 @@
     switch(force){ /* fall through each data type */
     default: skillsnice_usage();
     case 0: /* not forced */
+      if (argptr && argptr[0] == '-') /* its the next argument not a parameter */
+        continue;
     case 't':
       if(argc){
         struct stat sbuf;
